pkg/security/securitycontextconstraints/util/sort/bypriority.go

```diff
@@ -1,12 +1,14 @@
-package securitycontextconstraints
+package sort
```

Comment:
Doing so would cause an import cycle, as the `github.com/openshift/origin/pkg/security/apis/security/v1` package imports `github.com/openshift/origin/pkg/security/securitycontextconstraints/util`, which makes use of the security/v1 package when converting internals to v1